components/ParameterForm.vue

```diff
@@ -118,26 +128,32 @@ const renderAsRadios = (parameter: Parameter) => {
   return parameter.parameterType === ParameterType.Select && optionsAreTerse(parameter);
 };
 
-const submitForm = () => {
-  // Not implemented yet
-};
+const formSubmitting = ref(false);
+
+const submitForm = async () => {
+  if (!formData.value) {
+    return;
+  };
 
-const largeScreen = ref(true);
-const breakpoint = 992; // CoreUI's "lg" breakpoint
-const setFieldSizes = () => {
-  if (window.innerWidth < breakpoint) {
-    largeScreen.value = false;
-  } else {
-    largeScreen.value = true;
-  }
+  formSubmitting.value = true;
+  const response = await $fetch<NewScenarioData>("/api/scenarios", {
```

Comment:
I was imagining all of the scenario-related endpoints living under a single 'scenarios' namespace, rather than varying between 'scenario' and 'scenarios' depending on the use case. I believe in CRUD it is conventional to standardize on the plural form in route names.

Reply:
Yeah, it's a bit uneasy. Fair enough!